src/components/InvitationModal/index.tsx

```diff
@@ -0,0 +1,161 @@
+    ['goodEmail@gmail.com', 'Accepted'],
+    ['user1@example.com', 'Pending'],
+    ['user2@example.com', 'Accepted'],
+    ['ReallyLongNameThatWillNotFitInRowAbove@example.com', 'Accepted'],
+    ['goodEmail@gmail.com', 'Accepted'],
+  ]);
+
+  // Array to test invalid email
+  const validUsers = ['user1@example.com', 'user2@example.com'];
+  const [input, setInput] = useState('');
+  let valid = false;
+  const [validMessage, setValidMessage] = useState('');
+  const [validClassName, setValidClassName] = useState('');
+
+  function verifyUser(): void {
+    validUsers.forEach((element) => {
+      if (element === input) {
+        valid = true;
+        setValidMessage('Successfully sent!');
+        setValidClassName('valid-email');
+        setInput('');
+      }
+      if (!valid) {
+        valid = false;
+        setValidMessage('Invalid Email');
+        setValidClassName('invalid-email');
+      }
+    });
+  }
+
+  return (
+    <div className="invitation-modal-content">
+      <h2>Share Schedule</h2>
+      <p>
+        Enter the email associated with another user&apos;s GT-Scheduler account
+        & we&apos;ll them an invite via email to import this schedule into their
+        view
+      </p>
+      <div className="email-block">
+        {/* <form>
+                    <text className='invalid-email'>Invalid Email</text>
+                    <input
+                        type="email"
+                        id="email"
+                        className="email"
+                        placeholder="recipient@example.com"
+                    />
+                </form> */}
+        <div className="email-input">
+          <input
+            type="email"
+            id="email"
+            value={input}
+            className="email"
+            placeholder="recipient@example.com"
+            list="recent-invites"
+            onChange={(e): void => setInput(e.target.value)}
+          />
+          <datalist id="recent-invites">
+            {emails.map((element) => (
+              <option>{element[0]}</option>
+            ))}
+          </datalist>
+          <text className={validClassName}>{validMessage}</text>
+        </div>
+        <button
+          type="button"
+          className="send-button"
+          onClick={(e): void => {
+            verifyUser();
+          }}
+        >
+          Send Invite
+        </button>
+      </div>
+      <hr className="divider" />
+      <p>
+        Users Invited to View <strong>Primary</strong>
+      </p>
+      <div className="shared-emails">
+        {emails.map((element) => (
+          <div className="email-and-status">
+            <div className="individual-shared-email" id={element[1]}>
+              {element[0]}
+              <Button className="button-remove">
+                <FontAwesomeIcon className="circle" icon={faCircle} />
+                <FontAwesomeIcon className="remove" icon={faClose} />
+              </Button>
```

Comment:
Let's make use of some existing styles in GTS. There's a variable called `card` defined in `src/variables.scss`  that is used to make chips like in the image below. Check out how these are used in `CourseAdd` and `EventAdd`. I think with some minor extra styling, the chips would look better and we can remove some hard-code styling.

![image](https://user-images.githubusercontent.com/38119460/219935148-71d7bcbf-8da9-4142-b790-38e49c049cc3.png)